src/modules/launcher/PowerLauncher/MainWindow.xaml

```diff
@@ -27,69 +28,73 @@
     Visibility="{Binding MainWindowVisibility, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
     WindowStartupLocation="Manual"
     WindowStyle="None"
-    mc:Ignorable="d"
-    MinHeight="0">
+    mc:Ignorable="d">
 
     <Grid x:Name="RootGrid" MouseDown="OnMouseDown">
         <!--  We set the background here because the Acrylic can be too translucent / background too bright on Light theme  -->
         <Grid.Background>
             <SolidColorBrush Opacity="0.8" Color="{DynamicResource ApplicationBackgroundColor}" />
         </Grid.Background>
         <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" MaxHeight="{Binding Results.MaxHeight}" />
         </Grid.RowDefinitions>
         <Border
             x:Name="SearchBoxBorder"
             Grid.Row="0"
-            Margin="12,12,12,0"
-            Background="{DynamicResource LayerFillColorDefaultBrush}"
-            BorderBrush="{DynamicResource SurfaceStrokeColorDefaultBrush}"
-            BorderThickness="1"
-            CornerRadius="8">
+            Padding="12,4,12,3">
             <local:LauncherControl x:Name="SearchBox" />
         </Border>
         <local:ResultList
             x:Name="ListBox"
-            Grid.Row="1"
+            Grid.Row="2"

```

Comment:
I would make sense to move this below the KeywordsOverviewGrid :)

Reply:
Done!